.circleci/config.yml

```diff
@@ -89,11 +92,13 @@ jobs:
               A up;
               A run monitoring up;
               A down;
-              cd "$PROTOCOL_DIR";
+              cd audius-protocol/;
               git checkout -f;
               exit;
             EOF
 
+            echo "got status code from ssh agent" $?
```

Comment:
Here we're only echoing the status, but not stopping, nor failing, the build.

Reply:
good catch with the echo. i'm gonna remove all the echos

as for the status echo, i believe circle stops if the exit status of a command is not 0, i think it implicitly has a set -e. thats why there's some || true in the code